### DISCUSSION TITLE
Soft delete documentation clarification

### DISCUSSION THREAD

**Discussion by ljuba-roofr** [cycle/orm#498] [Q&A]
I think [2.x DeletedAt (SoftDelete)](https://cycle-orm.dev/docs/entity-behaviors-soft-delete/current/en) could use a bit more clarification. I'm using

```
"cycle/annotated": "^4.1",
"cycle/entity-behavior": "^1.3",
"cycle/orm": "^2.7",
"cycle/schema-builder": "^2.8",
```

I followed steps from [2.x DeletedAt (SoftDelete)](https://cycle-orm.dev/docs/entity-behaviors-soft-delete/current/en) docs page and I added `#[Behavior\SoftDelete` to the entity. I also added `deleted_at` column in postgresql that's set to timestamp. Since this page doesn't specify any other steps required I would expect this to be enough to get the soft deleted functionality working but when I query entities from that table I'm still getting records where `deleted_at` is set to a date

I looked a bit more in the docs and I found [1.x Soft Deleted Entities](https://cycle-orm.dev/docs/advanced-soft-deletes/1.x/en#scope) that shows this

> The soft deletion functionality can be achieved by applying a custom delete strategy in the mapper, combined with a global entity scope to limit the selection

Only when I added [1.x NotDeletedScope scope](https://cycle-orm.dev/docs/advanced-soft-deletes/1.x/en#scope) queries stopped returning soft deleted rows

Then I found [2.x NotDeletedScope scope](https://cycle-orm.dev/docs/advanced-scope/current) example but it's not as clear as `1.x` that this is required for soft delete behaviour or that I should add a mapper

Maybe [1.x Soft Deleted Entities](https://cycle-orm.dev/docs/advanced-soft-deletes/1.x/en#scope) page could be added in `2.x` docs and also linked in the [2.x DeletedAt (SoftDelete)](https://cycle-orm.dev/docs/entity-behaviors-soft-delete/current/en) page?

**Comment by roxblnfk** [comment]
Hello. I think it makes sense.
Would you like to create a PR?
